src/components/UI/DialogBox/DialogBox.module.css

```diff
@@ -40,6 +40,9 @@
   color: #073f24;
   padding-top: 24px !important;
   padding-bottom: 8px !important;
+  justify-content: center !important;
+  display: flex !important;
+  flex-direction: column !important;
```

Comment:
There is a property in Dialog box that you can pass to make the title center aligned.

Reply:
@mdshamoon  could you elaborate on which property? I see the CSS written for `DialogBox` as follows:
```
.Dialogbox {
  max-width: 80%;
  min-width: 459px;
  margin: 0 auto;
  height: unset !important;
  max-height: 90% !important;
}
```
is it the `margin: 0 auto`